devops/scripts/vagrant_package.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+molecule test -s vagrant_packager && \
+# Unfortunately since we need to prompt the user for sudo creds..
+# I had to break the actual vagrant package logic outside of molecule
+molecule/vagrant_packager/package.py && \
+molecule destroy -s vagrant_packager
```

Comment:
Ha, I ran into a similar problem over in https://github.com/freedomofpress/ansible-role-grsecurity-build/pull/31, with `vars_prompt` failing to run in non-interactive mode. I wouldn't hesitate to put logic this simple directly in the Makefile, but no objections here.